composer.json

```diff
@@ -20,15 +20,15 @@
 	"require": {
 		"php": "^7.1",
 		"composer/installers": "*",
-		"openbuildings/purchases": "^0.11",
-		"clippings/freezable": "0.2.*",
-        "openbuildings/jam": "^0.5.13"
+		"openbuildings/purchases": "^0.12",
+		"clippings/freezable": "0.3",
```

Comment:
could this be ^0.3 in order to cover future minor updates

Reply:
Good catch. Thanks!